zubhub_frontend/zubhub/src/views/project_details/projectDetailsScripts.js

```diff
@@ -135,3 +135,15 @@ export const isGdriveORVimeoORYoutube = url => {
     return false;
   }
 };
+
+export const handleMobileShare = async (project) => {
+  const shareData = {
+    title: project.title,
+    text: "Check out this amazing Zubhub project! ",
+    url: ""
+  }
+  try {
+    await navigator.share(shareData);
+  } catch (err) {
+  }
```

Comment:
Maybe we could use a Toast like some of the other pages do when an error occurs to let the user know.